src/doc/howtotestsharedrocache.txt

```diff
@@ -0,0 +1,26 @@
+0. setup a parent image and a clone image
```

Comment:
Definitely would need a real RST-based doc section added to the RBD docs.

Reply:
> Good start -- can you also add some googlemock test cases to "src/test/librbd/cache/test_mock_ParentImageCache.cc```?

 i have followed your UT and mock to write some test cases for SharedReadOnlyCache at src/test/librbd/cache/test_mock_ParentImageCache.cc. And cover all interface of SharedReadOnlyCache.